### PR TITLE
fix(config): replace systray icon

### PR DESCRIPTION
**Pull request by MCozhusheck**
Description
---
Replace system tray icon according to designs.

Motivation and Context
---
Keep design according to the figma.

How Has This Been Tested?
---
Just run app and check menu.

What process can a PR reviewer use to test or verify this change?
---
same as above.

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

**Comment by MCozhusheck** [comment]
There issue on dark theme - the contrast with logo with menu bar is too low.
AFAIK system support for themed tray icons is only in Tauri V2 but there is [work around on V1](https://github.com/tauri-apps/tauri/issues/6645#issuecomment-1981537369)

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -69,12 +67,12 @@
             "identifier": "com.tari.universe",
             "icon": [
                 "icons/icon.icns",
-                "icons/icon.ico",
```
Comment (stringhandler) [inline src-tauri/tauri.conf.json:72]: is this not needed for windows?

**Comment by stringhandler** [comment]
Needs testing on windows

**Comment by stringhandler** [comment]
fails to build on windows
```
error: failed to run custom build command for `tari-universe v0.2.8 (D:\projects\tari-universe\src-tauri)`
note: To improve backtraces for build dependencies, set the CARGO_PROFILE_DEV_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.

Caused by:
  process didn't exit successfully: `D:\projects\tari-universe\src-tauri\target\debug\build\tari-universe-f50d4c0f22370370\build-script-build` (exit code: 1)
  --- stdout
  cargo:rerun-if-env-changed=TAURI_CONFIG
  ```

**Comment by stringhandler** [comment]
also getting compile errors on mac

**Comment by stringhandler** [comment]
changing milestone to v0.3

**Pull request closed without merging**